com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ActivitiExceptionHandler.java

```diff
@@ -0,0 +1,77 @@
+package com.sap.cloud.lm.sl.cf.process.util;
+
+import java.sql.Timestamp;
+import java.text.MessageFormat;
+
+import org.activiti.engine.delegate.event.ActivitiEvent;
+import org.activiti.engine.delegate.event.ActivitiExceptionEvent;
+import org.activiti.engine.history.HistoricVariableInstance;
+import org.activiti.engine.impl.persistence.entity.VariableInstance;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sap.cloud.lm.sl.common.SLException;
+import com.sap.cloud.lm.sl.persistence.model.ProgressMessage;
+import com.sap.cloud.lm.sl.persistence.model.ProgressMessage.ProgressMessageType;
+import com.sap.cloud.lm.sl.persistence.services.ProgressMessageService;
+
+public class ActivitiExceptionHandler {

```

Comment:
The class handles ActivitiExceptionEvent. Not Exceptions thrown by the Activiti :)